src/uu/wc/src/wc.rs

```diff
@@ -592,8 +607,10 @@ fn word_count_from_input(input: &Input, settings: &Settings) -> CountResult {
 /// is close enough to pass the GNU test suite.
 fn compute_number_width(inputs: &[Input], settings: &Settings) -> usize {
     if inputs.is_empty()
-        || (inputs.len() == 1 && settings.number_enabled() == 1)
-        || (settings.files0_from_stdin_mode && settings.number_enabled() == 1)
+        || settings.number_enabled() == 1
+            && (inputs.len() == 1
+                || settings.files0_from_path.as_ref().map(|p| p.as_os_str())
+                    == Some(OsStr::new(STDIN_REPR)))
```

Comment:
Oof this formatting is a bit rough. Is that really what rustfmt wants? Maybe you could break up into a few smaller booleans first?